.circleci/config.yml

```diff
@@ -1,92 +1,87 @@
-version: 2
-jobs:
-  deploy_api:
-    docker:
-      -
+jobs: 
+  deploy_api: 
+    docker: 
+      - 
         image: docker
-    environment:
-      -
-        VERSION: "0.1.2"
-    steps:
-      -
-        checkout:
+    steps: 
+      - checkout: 
           path: ~/repo
-      -
-        attach_workspace:
+      - attach_workspace: 
           at: /tmp/workspace
       - setup_remote_docker
-      -
-        run:
+      - run: 
           command: |
               mkdir ~/repo/api/dist
```

Comment:
This whole section might be more palatable if we put it in a bash file in `etl/scripts`?